src/Console/Command/CleanBranchCommand.php

```diff
@@ -137,7 +138,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 $notfound[] = $dir;
             }
 
-            $progressBar->setProgress($index);
+            $progressBar->setProgress((int) $index);
```

Comment:
This might not work. Since `$index` here is always the path of the `$dir` itself.
So you might want to replace this with `$progressBar->advance()`